drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -11,7 +11,9 @@
 RSpec.describe Hmis::Form::FormProcessor, type: :model do
   include_context 'hmis base setup'
 
-  let!(:fd) { create :hmis_form_definition }
+  let(:fd) { Hmis::Form::Definition.find_by!(role: :INTAKE) }
+  let(:fd_exit) { Hmis::Form::Definition.find_by!(role: :EXIT) }
+  let(:fd_post_exit) { Hmis::Form::Definition.find_by!(role: :POST_EXIT) }
```

Comment:
Thanks for updating the spec! Sorry there was so much churn. 

Shouldn't there be a test to check that hud_values which are not present in the form definition mappings result in an error or rejection? I don't see that test case here (but I might be missing it in this monster spec). I hate to recommend adding more code to this but I do think it would be a worthwhile to add a minimal test to ensure the restriction is applied.

Reply:
yeah I will add for sure